frontend/app/templates/create-badges.hbs

```diff
@@ -1,198 +1,199 @@
+            </div>
+            <div class="content">
+               <div class="center aligned">
+                  {{ background-component/color-bg-component sendDefColor=(action 'mutateDefColor') backColor=backColor }}
+               </div>
+            </div>
+         </div>
+      </div>
+      <div class="ui segment">
+         <div class="ui form width-container">
+            <p>
+               <b>Select from following font customizations</b>
+            </p>
+            <div class="ui horizontal divider"></div>
+            <div class="content">
+               <div class="center aligned">
+                  {{ text-component/font-color sendDefFontColor=(action 'mutateDefFontColor') fontColor=fontColor}}
+               </div>
+            </div>
+            <div class="ui horizontal divider"></div>
+            <div class="content">
+               <div class="center aligned">
+                  {{ text-component/font-size fontsizeall=fontsizeall fonttypeall=fonttypeall sendDefSize=(action 'mutateFontSize') sendDefFont=(action 'mutateCustomFont')}}
+               </div>
+            </div>
+         </div>
+      </div>
+      <button {{action "submitForm"}} class="ui fluid orange {{if authToken.isEmailVerified '' 'disabled'}} button">Create Badges</button>
+      {{#each userError as |error|}}
+      <p>{{error.message}}</p>
+      {{/each}}
+      {{#if showProgress}}
+      <div class="ui hidden divider"></div>
+      <div>
+         <div class="ui centered aligned grid">{{progressState}}</div>
+         <div class="ui hidden divider"></div>
+         {{#ui-progress percent=progress class="teal indicating"}}
+         <div class="bar">
+            <div class="progress"></div>
+         </div>
+         {{/ui-progress}}
+      </div>
+      {{/if}}
+      {{#if badgeGenerated}}
+      <div class="ui positive message">
+         <i class="close icon"></i>
+         <div class="ui center aligned header">
+            Badge generated successfully!
+         </div>
+         <div class="ui horizontal divider"></div>
+         <div class="ui center aligned secondary header">
+            <a class="ui green basic button" href="{{badgeGeneratedLink}}">
+            Download as PDF
+            </a>
+         </div>
+         <script type="text/javascript">
+            $('.message .close')
+                .on('click', function () {
+                    $(this)
+                        .closest('.message')
+                        .transition('fade')
+                        ;
+                });
+         </script>
+      </div>
+      {{/if}}
+   </div>
+</div>
+<div class="{{if previewToggled 'six' 'hide'}} wide column preview">
+   <!--Add Preview Section here-->
+   {{
+   user-component/preview
+   fn=firstName
+   ln=lastName
+   org=organization
+   sh=socialHandle
+   desg=designation
+   defColor=defFontColor
+   font_fn=defFont1Size
+   font_ln=defFont2Size
+   font_desg=defFont3Size
+   font_org=defFont4Size
```

Comment:
Why was these changes done?
There is a deintendation of 1 space I guess. Maybe you could restore that particular commit and then intend those 4 lines.